### PR TITLE
Contributing mention enrolling keys (Infra)

### PR DESCRIPTION
**Pull request by kissiel**
## Description
Add paragraph about keys enrollment to GitHub in CONTRIBUTING.md.
Fix syntax and formatting problems in CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -116,14 +126,18 @@ use the standard [unittest library].
 Ensure the job and test plan definitions follow the correct syntax using
 the `validate` command:
 
-    $ ./manage.py validate
+```bash
+    ./manage.py validate
```
Comment (Hook25) [inline CONTRIBUTING.md:130]: Uniformity, remove space?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -116,14 +126,18 @@ use the standard [unittest library].
...
+```
 
 ### Writing and running unit tests for providers
 
 Run checks for code quality of provider hosted scripts and any unit
 tests for providers:
 
-    $ ./manage.py test
+```bash
+    ./manage.py test
```
Comment (Hook25) [inline CONTRIBUTING.md:139]: Uniformity, remove space?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -272,13 +290,16 @@ repository] first.
...
+    ```bash
+    git checkout -b 123456-invalid-session-content
+    ```
 
 1. Work on your changes, test them, iterate, commit your work.
 
 1. Before sending your changes for review, make sure to rebase your work using
    the most up-to-date data from the main repository:
 
+    ```bash
```
Comment (Hook25) [inline CONTRIBUTING.md:302]: Uniformity, remove space?

**Review by Hook25 — APPROVED** [review]
A few comments here and there, but overall nice contribution. Reading this I'm wondering if we should have a better way to direct contributors to this page when the change is not signed (the error message is a bit difficult to read)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -43,24 +51,24 @@ install everything you need in a Python virtual environment.
...
 
 Prepare the development environment. If you are an external contributor and
 plan on submitting some changes, you will have to [fork the Checkbox repository
 first], and clone your own version locally. Otherwise:
 
 ``` bash
-$ cd ~
-$ git clone git@github.com:canonical/checkbox.git
+cd ~
+git clone git@github.com:canonical/checkbox.git
```
Comment (pieqq) [inline CONTRIBUTING.md:63]: I originally wrote code snippets like this, but @tang-mm pointed out this makes it harder to see what lines are commands and what lines are output from the command.

Something like:

```
$ ls
AccountsService/      git/                    snapd/
alsa/                 grub/                   snmp/
$ cd git
$ ls -l
total 0
```

is easier to read than

```
ls
AccountsService/      git/                    snapd/
alsa/                 grub/                   snmp/
cd git
ls -l
total 0
```

(I know it's not the case in this change, but there might be some commands with an output in this doc at some point)

**Review by pieqq — CHANGES_REQUESTED** [review]
Just a quick comment regarding code snippets.

File: CONTRIBUTING.md (hunk 4 above)
Comment (kissiel) [reply CONTRIBUTING.md:63]: And I 100% agree with that, the problem is that the thing I removed is:
```
$ cd ~/checkbox/checkbox-ng
$ ./mk-venv
$ . ~/checkbox-ng/venv/bin/activate
```
In which there's no output of the commands being run, thus making it not really the case (and also a MD syntax problem).

File: CONTRIBUTING.md (hunk 3 above)
Comment (kissiel) [reply CONTRIBUTING.md:302]: but this codeblock has to be a part of the numbered list item.

**Comment by kissiel** [comment]
In the meantime most of my contribution covered by this PR landed with other PRs.
Closing this gig.

**Pull request closed without merging**